kml/serdes_gpx.cpp

```diff
@@ -0,0 +1,273 @@
+#include "kml/serdes_gpx.hpp"
+
+#include "indexer/classificator.hpp"
+
+#include "coding/hex.hpp"
+#include "coding/point_coding.hpp"
+#include "coding/string_utf8_multilang.hpp"
+
+#include "geometry/mercator.hpp"
+
+#include "base/assert.hpp"
+#include "base/stl_helpers.hpp"
+#include "base/string_utils.hpp"
+#include "base/timer.hpp"
+
+#include <sstream>
+
+using namespace std::string_literals;
```

Comment:
Can we use std::string_view instead? In any case, using namespace should be moved to the internal gpx namespace.

Reply:
Switched to string_view